### ISSUE TITLE
`total_num_canonical_blocks` is wrong

### ISSUE THREAD

**Issue by n1tranquilla** [Granola-Team/mina-indexer#1557]
Here is a query where we are limiting to the first 6k blocks, however the result for `total_num_blocks` is the length of the entire blockchain.

Query:
```
query BlocksQuery($query: BlockQueryInput!, $limit: Int = 10, $sort_by: BlockSortByInput!) {
  blocks(query: $query, limit: $limit, sortBy: $sort_by) {
    total_num_blocks
    total_num_canonical_blocks
    total_num_supercharged_blocks
    blockHeight
    globalSlotSinceGenesis
    transactions {
      coinbase
    }
    snarkFees
    txFees
    canonical
  }
}
```

Variables:
```
{
  "limit": 10000000,
  "sort_by": "BLOCKHEIGHT_DESC",
  "query": {
    "blockHeight_lte": 6000
  }
}
```

Response:
```
{
  "data": {
    "blocks": [
      {
        "total_num_blocks": 359614,
        "total_num_canonical_blocks": 359604,
        "total_num_supercharged_blocks": 142247,
        "blockHeight": 6000,
        "globalSlotSinceGenesis": 8478,
        "transactions": {
          "coinbase": "1440000000000"
        },
        "snarkFees": "0",
        "txFees": "1000000",
        "canonical": true
      },
...
```

**Comment by n1tranquilla** [comment]
The data is probably right, just not presented correctly or in the right place. We need to rethink our `total_num_*` implementations and see if they belong somewhere else.

**Comment by Isaac-DeFrain** [comment]
This datum is about the total number of canonical blocks, similarly for the others. What are you expecting? Number of (canonical/supercharged/etc) blocks bounded by the query?

**Comment by Isaac-DeFrain** [comment]
These `total_num_*` quantities probably belong in the summary

**Comment by n1tranquilla** [comment]
> These total_num_* quantities probably belong in the summary

Yes, as a user in the front end, I expected the query for that block height to give me the correct value for `total_num_*` and was surprised by the result, so I think your thoughts referenced above are on the right track. 

I'll assign back to you for now.

**Comment by Isaac-DeFrain** [comment]
Okay. This sort of response requires some additional support in the database. We'll need to keep CFs for `block height -> number of * blocks at that height` in order to answer the general query.

**Comment by Isaac-DeFrain** [comment]
@n1tranquilla where does this issue stand?

**Comment by n1tranquilla** [comment]
This is a presentation issue with the data. It's not strictly wrong, it's just presented in a way that it might be used improperly. I think we need to deprioritize this issue. Moving to bottom of backlog.

**Comment by n1tranquilla** [comment]
This field can be deprecated. Perhaps mark as deprecated. It is no longer used in FE

**Comment by Isaac-DeFrain** [comment]
https://github.com/Granola-Team/mina-indexer/commit/184d2ec2ad65f321792d5c592ae85ebb370a8d84 .. https://github.com/Granola-Team/mina-indexer/commit/18b039b9a5005f6f3e5f5ff4327f4eb8c9ae115d and https://github.com/Granola-Team/mina-indexer/commit/c6745e86725b5fc80300ef442cfac8d70b41d39c
